docs/howtos/create_custom_appcharts.md

```diff
@@ -21,6 +21,10 @@ To add a custom Helm chart to the system, there are two mandatory steps:
 
 The following two sections explain these steps in detail.
 
+An advanced feature of custom appliction charts is the ability to expose user-settable configuration
```

Comment:
small typo in `appliction`. Should be `application`

Reply:
Huh. Not just here. Quite a number of other places too.

Reply:
Done with commit 4d761fdddbb4caaa7406d61d62e536492134792c